fdbrpc/FlowTransport.h

```diff
@@ -164,6 +170,9 @@ class FlowTransport {
 	std::map<NetworkAddress, std::pair<uint64_t, double>>* getIncompatiblePeers();
 	// Returns the same of all peers that have attempted to connect, but have incompatible protocol versions
 
+	Future<Void> onIncompatibleChanged();
+	// Returns when getIncompatiblePeers has at least one peer which is incompatible.
```

Comment:
VS code (or other IDE) would prefer comment before declaration, otherwise the tooltip for this function becomes the one for `getIncompatiblePeers()`. Sadly, this style is used a lot in this file (and other files).